### ISSUE TITLE
"peewee.OperationalError: no such function: REGEXP" when trying to check in

### ISSUE THREAD

**Issue by YannCNovelli** [codito/habito#14]
First, thank you for this promising tracker. 
However, I'm getting an error when trying to check in.

What I typed:
`habito checkin anki -q 1.0`

What I got:
```
Traceback (most recent call last):
  File "/home/datasc/.local/lib/python3.5/site-packages/peewee.py", line 2484, in execute_sql
    cursor.execute(sql, params or ())
sqlite3.OperationalError: no such function: REGEXP

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/datasc/.local/bin/habito", line 11, in <module>
    sys.exit(cli())
  File "/home/datasc/.local/lib/python3.5/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/datasc/.local/lib/python3.5/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/datasc/.local/lib/python3.5/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/datasc/.local/lib/python3.5/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/datasc/.local/lib/python3.5/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/datasc/.local/lib/python3.5/site-packages/habito/habito.py", line 147, in checkin
    if habits.count() == 0:
  File "/home/datasc/.local/lib/python3.5/site-packages/peewee.py", line 1447, in inner
    return method(self, database, *args, **kwargs)
  File "/home/datasc/.local/lib/python3.5/site-packages/peewee.py", line 1705, in count
    return Select([clone], [fn.COUNT(SQL('1'))]).scalar(database)
  File "/home/datasc/.local/lib/python3.5/site-packages/peewee.py", line 1447, in inner
    return method(self, database, *args, **kwargs)
  File "/home/datasc/.local/lib/python3.5/site-packages/peewee.py", line 1691, in scalar
    row = self.tuples().peek(database)
  File "/home/datasc/.local/lib/python3.5/site-packages/peewee.py", line 1447, in inner
    return method(self, database, *args, **kwargs)
  File "/home/datasc/.local/lib/python3.5/site-packages/peewee.py", line 1678, in peek
    rows = self.execute(database)[:n]
  File "/home/datasc/.local/lib/python3.5/site-packages/peewee.py", line 1447, in inner
    return method(self, database, *args, **kwargs)
  File "/home/datasc/.local/lib/python3.5/site-packages/peewee.py", line 1518, in execute
    return self._execute(database)
  File "/home/datasc/.local/lib/python3.5/site-packages/peewee.py", line 1672, in _execute
    cursor = database.execute(self)
  File "/home/datasc/.local/lib/python3.5/site-packages/peewee.py", line 2497, in execute
    return self.execute_sql(sql, params, commit=commit)
  File "/home/datasc/.local/lib/python3.5/site-packages/peewee.py", line 2491, in execute_sql
    self.commit()
  File "/home/datasc/.local/lib/python3.5/site-packages/peewee.py", line 2287, in __exit__
    reraise(new_type, new_type(*exc_args), traceback)
  File "/home/datasc/.local/lib/python3.5/site-packages/peewee.py", line 170, in reraise
    raise value.with_traceback(tb)
  File "/home/datasc/.local/lib/python3.5/site-packages/peewee.py", line 2484, in execute_sql
    cursor.execute(sql, params or ())
```
As a quick and dirty fix I changed line 148 
`habits = HabitModel.select().where(HabitModel.name.regexp(query))` 
in 
`~/.local/lib/python3.5/site-packages/habito/habito.py` 
to 
`habits = HabitModel.select().where(HabitModel.name==query)` in order to skip the regex query entirely and query the exact habit name instead.

**Comment by codito** [comment]
@YannCNovelli by any chance do you have `peewee >= 3.x` installed?

**Comment by YannCNovelli** [comment]
@codito That's correct. I have version 3.0.17.

**Comment by codito** [comment]
Fixed in `v1.0a5` version on pypi. Please upgrade and try it.

**Comment by YannCNovelli** [comment]
That fixed it. Thanks!

**Comment by bolorundurovj** [comment]
> Fixed in `v1.0a5` version on pypi. Please upgrade and try it.

do you mean peewee version 1.0a5?
